src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1219,9 +1217,24 @@ int fromCommandLine (
         nRet                    = rpcINTERNAL;
     }
 
-    std::cout << jvOutput.toStyledString ();
+    return { nRet, std::move(jvOutput) };
+}
+
+//------------------------------------------------------------------------------
+
+namespace RPCCall {
+
+int fromCommandLine (
+    Config const& config,
+    const std::vector<std::string>& vCmd,
+    Logs& logs)
+{
+    auto const result = rpcClient(vCmd, config, logs);
+
+    if (result.first != 1)
```

Comment:
I'd replace `1` with a symbolic constant, or at least add a comment here documenting that `1` means `print_usage`